src/main/java/com/internet/shop/controllers/shopping/carts/ShoppingCartController.java

```diff
@@ -0,0 +1,26 @@
+package com.internet.shop.controllers.shopping.carts;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ShoppingCartController extends HttpServlet {
```

Comment:
let's add verbs to Controller's names for example  GetAllUsersController (here and everywhere)